src/Microsoft.ML.Transforms/NAIndicatorTransform.cs

```diff
@@ -4,25 +4,35 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Data.Conversion;
 using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
 
-[assembly: LoadableClass(typeof(NAIndicatorTransform), typeof(NAIndicatorTransform.Arguments), typeof(SignatureDataTransform),
-   NAIndicatorTransform.FriendlyName, "NAIndicatorTransform", "NAIndicator", NAIndicatorTransform.ShortName, DocName = "transform/NAHandle.md")]
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(IDataTransform), typeof(NAIndicatorTransform), typeof(NAIndicatorTransform.Arguments), typeof(SignatureDataTransform),
+    NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoadName, "NAIndicator", NAIndicatorTransform.ShortName, DocName = "transform/NAHandle.md")]
 
-[assembly: LoadableClass(typeof(NAIndicatorTransform), null, typeof(SignatureLoadDataTransform),
-    NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoaderSignature)]
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(IDataTransform), typeof(NAIndicatorTransform), null, typeof(SignatureLoadDataTransform),
+    NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoadName)]
+
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(NAIndicatorTransform), null, typeof(SignatureLoadModel),
+    NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoadName)]
+
+[assembly: LoadableClass(typeof(IRowMapper), typeof(NAIndicatorTransform), null, typeof(SignatureLoadRowMapper),
+   NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoadName)]
 
 namespace Microsoft.ML.Runtime.Data
 {
     /// <include file='doc.xml' path='doc/members/member[@name="NAIndicator"]'/>
-    public sealed class NAIndicatorTransform : OneToOneTransformBase
+    public sealed class NAIndicatorTransform : OneToOneTransformerBase
     {
         public sealed class Column : OneToOneColumn
```

Comment:
>public [](start = 8, length = 6)

as we go through this, if you have time to add a one line comment on the public members, a they will appear in the documentation.

Reply:
Ok will do that!

---
In reply to: [224315913](https://github.com/dotnet/machinelearning/pull/1217#discussion_r224315913) [](ancestors = 224315913)